### ISSUE TITLE
Mobile styles

### ISSUE THREAD

**Issue by mvultgmr** [ThePix/QuestJS#23]
Hi, Pix.  I had some ideas for adding mobile breakpoints to style.css, so I forked the project and did some revisions.  I'm new to GitHub so am not sure if this is the right way to do things, but here is a link to my version.  Feel free to play with!

https://github.com/mvultgmr/QuestJS/commit/c4079a2ddd4664aa5661c353788babfae3e73252

**Comment by ThePix** [comment]
Just to let you know, I am thinking about this. I think it is important, I am not sure the best way to proceed. Part of that is that the author should have some choice about whether the side pane (and anything else, map?) disappears, depending on what the authors thinks is the primary way to play the game.

**Comment by mvultgmr** [comment]
Yeah, I wasn't sure immediately what to do with the side pane and map items. The solution usually is to move all that to a hamburger menu; perhaps something for you to think about. In the meantime, you might preserve this code as a starting point for authors who want to attempt it, either as commented-out code or within a help file. 

I'm glad the project seems to be going well!

**Comment by ThePix** [comment]
I have made a small step, it that now features disappear if the width is narrow.
https://github.com/ThePix/QuestJS/wiki/Responsive-Games

May need more consider, eg your hamburger idea; we will see how it goes.

**Comment by mvultgmr** [comment]
Great job. Mobile styling is a tedious bitch, but having these features will make things so much more accessible.

Also great job on the map function!

**Comment by KVonGit** [comment]
I tried to check out the code by @mvultgmr, but the link was dead.

One of the things on my QuestJS to-do list is to add the hamburger-style menu for the panes when on mobile.

Also, I wanted to see if those mobile settings were better than my mobile settings (which are rudimentary, at best).

**Comment by mvultgmr** [comment]
Shit, sorrry, @KVonGit . I purged the repo once I thought Pix was done with it. But if you'd like to share your code, I'd be be happy to spitball it.  :)

**Comment by mvultgmr** [comment]
Here is the mobile styling code that I wrote last October, FWIW.  I believe a the developer has devised better options for mobile styling since then.
[style-mobile.txt](https://github.com/ThePix/QuestJS/files/6678879/style-mobile.txt)

**Comment by KVonGit** [comment]
To avoid creating a duplicate issue, I shall necrobump this one. =)

In order to get the `@media...` CSS stuff to work, I had to add this to the `head` element in my **index.html** file:
```html
<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

---
And all I've added so far (besides `settings.panesCollapseAt = 953`) is:

```css
@media only screen and (max-width: 953px) 
{
  #main{
    margin-left: 0 !important;
    margin-right: 0 !important;
  }
  .object-link{
    font-size: inherit !important;
  }
}
```

---
I need to look at the settings in this last post, and most probably add stuff from that, too.

If anyone would like to see a game with these changes, it can be played here via GitHub Pages: https://kvongit.github.io/Koww_QuestJS/play.html
